package.xml

```diff
@@ -19,6 +19,8 @@
   <build_depend>eigen</build_depend>
   <build_depend>eigen_stl_containers</build_depend>
   <build_depend>libconsole-bridge-dev</build_depend>
+  <build_depend condition="$ROS_DISTRO != noetic">libfcl-dev</build_depend>
```

Comment:
does this include apply for ROS2 as well? or should it be `==melodic` else ...

Reply:
no idea how ROS 2 handles this... I haven't dived into it yet... so maybe just melodic/noetic would be better

Reply:
Debian bullseye contains 0.6.1, debian sid 0.7.1(and therefore also in Ubuntu 22.04) and there was no ros based releases except for noetic so this sounds like the correct approach